verticapy/ml_pipeline/README.md

```diff
@@ -0,0 +1,224 @@
+# HEADER
+---
+Vertica Pipelines is an open source platform for managing data scientists' machine learning pipelines. They are built on a human-readable data format: **YAML**.   
+
+## GETTING STARTED  
+This tutorial will be done in a Linux/Unix/Ubuntu environment.  
+### Install Verticapy  
+`$ pip install verticapy`  
+
+
+### Create Your first YAML file
+> **THE BASIC RULES OF ANY YAML FILE:**  
+> - YAML is case sensitive
+> - The files should have **.yaml** as the extension
+> - YAML does not allow the use of tabs while creating YAML files; use spaces instead
+
+The information in connect is the same you will use in Verticapy. An excellent tutorial of setting this up on the engdev machines can be found [**here**](https://confluence.verticacorp.com/pages/viewpage.action?pageId=140576924).  
+`$ vim connection.yaml`
+```yaml
+# Replace with your information #
+host: "10.20.50.86"
+port: "32802"
+database: "verticadb21455"
+password: ""
+user: "znowak"
```

Comment:
I would change all these to generics such as "your-username" and "database-name", etc.